.wpilib/wpilib_preferences.json

```diff
@@ -1,6 +1,6 @@
 {
     "enableCppIntellisense": false,
     "currentLanguage": "java",
-    "projectYear": "2023",
+    "projectYear": "2024beta",
```

Comment:
New year, new bot! Dunno why it says 'beta' in there though. Doing a search through the wpilibsuite in GitHub doesn't show me anything worthwhile (https://github.com/search?q=org%3Awpilibsuite+projectYear&type=code).

Meh, if this breaks something I guess we'll find out.

Reply:
Never mind. I didn't like the `beta` part so I removed it. It still builds though.